src/generate/project.rs

```diff
@@ -341,9 +341,9 @@ balance = 100_000_000_000_000
 # btc_address: mjSrB3wS4xab3kYqFktwBzfTdPg367ZJ2d
 
 [devnet]
-disable_bitcoin_explorer = true
-# disable_stacks_explorer = true
-# disable_stacks_api = true
+disable_stacks_explorer = true
+disable_stacks_api = true
```

Comment:
Is this intentional to disable the Stacks API and Explorer by default? I would expect the API is normally used, no?

Reply:
Addressed in latest